hera_mc/cm_hookup.py

```diff
@@ -38,11 +37,16 @@ def __init__(self, session=None):
             self.session = session
         self.handling = cm_handling.Handling(session)
         self.part_type_cache = {}
+        self.hookup_local_file = os.path.join(os.path.expanduser('~'), 'hookup_tmp.npy')
```

Comment:
Do Macs have a `~/.cache/` directory by default? I'd like to store this file as `~/.cache/hera_mc_hookup_cache.npy` or `~/.hera_mc_hookup_cache.npy` for clarity and to keep directory listings clean.

Also, you can jut write `os.path.expanduser('~/.hera_mc_hookup_cache.npy')` and it will work, rather than using the `join`.